app/grants/views.py

```diff
@@ -543,16 +543,6 @@ def get_grants(request):
     }
     _grants = get_grants_by_filters(**filters)
 
-    if sort == '' and keyword:
```

Comment:
@chibie @thelostone-mc I removed this because it seems like the same filter is being applied here? https://github.com/gitcoinco/web/blob/a881e53b349217d29b62c9b12f209d0a5e14a018/app/grants/views.py#L784

I changed this pr to draft because I'm not sure if my assumption is correct. Thanks!

Reply:
Ahh ok thanks! It looks like the query on L784 is also a case insensitive query 🚀  https://docs.djangoproject.com/en/4.0/ref/models/querysets/#icontains

Reply:
@schultztimothy

n/b:

+ removing L546 - L555 will break the exact title matches in grants explorer. shouldn't there be a replacement?

+ it doesn't hit the database as it only does a list comprehension on the _grants variable which contains a page of paginated data

Reply:
@chibie I ended up adding this back in. Just lmk if there is anything else!